packages/theme/src/button-group/adaptation.less

```
@@ -0,0 +1,35 @@
+/* stylelint-disable function-no-unknown */
+@import '../mixins/common.less';
+@import '../mixins/button-group.less';
+@import '../custom.less';
+@import './vars.less';
+
+@button-group-prefix-cls: ~'@{css-prefix}button-group';
+@group-item-prefix-cls: ~'@{css-prefix}group-item';
+@button-group-popover-prefix-cls: ~'@{css-prefix}group-item__more-popover';
+
+.@{button-group-prefix-cls} {
```

Stylelint (17.3.0)

[error] Unknown word button-group-prefix-cls (CssSyntaxError)

(CssSyntaxError)